libs/ui/src/ui_strings/audio_player.ts

```diff
@@ -14,15 +24,154 @@ export interface AudioPlayer {
   readonly stop: () => void;
 }
 
+/**
+ * ToneJS GrainPlayer configuration used for a particular playback rate.
+ *
+ * Audio grain configuration needs to be varied for slower playback rates to
+ * minimize phasing and echo artefacts. The values below have been tuned
+ * manually, but may need tweaking after some testing with a wider variety of
+ * clips.
+ *
+ * See https://en.wikipedia.org/wiki/Granular_synthesis for more background info
+ * on granular synthesis as a method of varying audio speed and pitch
+ * independently.
+ */
+interface RateBasedConfig {
+  grainSizeSeconds: number;
+  grainOverlapSeconds: number;
+}
+
+const RATE_CONFIG_VERY_SLOW: RateBasedConfig = {
+  grainOverlapSeconds: 0.025,
+  grainSizeSeconds: 0.02,
+};
+
+const RATE_CONFIG_SLOW: RateBasedConfig = {
+  grainOverlapSeconds: 0.05,
+  grainSizeSeconds: 0.1,
+};
```

Comment:
I see that this config is the same as the default config. Okay with keeping it to make fine tuning easier but also okay with dropping it to avoid confusion, and just keeping the very-slow and default configs

Reply:
Thanks! I'd missed the fact the last 2 configs had settled on the same params after tuning -- will drop this